yarnitor/background/worker.py

```diff
@@ -305,6 +307,9 @@ def _generate_listing(self):
             return {}
         apps = cluster_apps['apps']['app']
 
+        # Number of apps with unknown state
+        unknown_state = 0
```

Comment:
It looks like this isn't really used anywhere. Did something get left out of this code where unknown_state was used?